target/linux/ath79/dts/qca9531_glinet_gl-xe300-nor-nand.dts

```diff
@@ -0,0 +1,153 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca953x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "glinet,gl-xe300-nor-nand", "qca,qca9531";
+	model = "GL.iNet GL-XE300 Puli";
```

Comment:
Please decide for one name and use it consistently.
Are there other variants around?

Reply:
Ok I removed 'Puli', thanks.